tuf/client/updater.py

```diff
@@ -1541,18 +1548,17 @@ def _verify_uncompressed_metadata_file(self, metadata_file_object,
       None.
     """
 
-    metadata = metadata_file_object.read().decode('utf-8')
-    
+    metadata = metadata_file_object.read()
     try:
-      metadata_signable = tuf.util.load_json_string(metadata)
-    
+      metadata_signable = tuf.util.load_string(metadata) # TODO: <~> VERIFY THIS.
+
     except Exception as exception:
       raise tuf.InvalidMetadataJSONError(exception)
     
     else:
       # Ensure the loaded 'metadata_signable' is properly formatted.  Raise
       # 'tuf.FormatError' if not.
-      tuf.formats.check_signable_object_format(metadata_signable)
+      tuf.formats.check_signable_object_format(metadata_signable) # TODO: <~> This needs some way of knowing that it has a piece of DER-signed data so that it can re-encode 'signed' as DER before checking it.
```

Comment:
I agree. It's a habit I have for temporary comments that I need to resolve before the code reaches merge-worthy state. Same goes for the '<~>' mark: generally, these are things I do not want to allow in, and would grep for.